modules/rostests/apitests/win32nt/ntuser/NtUserSetTimer.c

```diff
@@ -7,12 +7,280 @@
 
```

Comment:
The code header needs to be adjusted accordingly (@binarymaster could help you with that)

Reply:
Thanks, I hope I fixed the style. I'll look at the header in other files tomorrow, if you can give me some advice I'll be glad(@binarymaster).

Reply:
Updated.